lib/composer

```diff
@@ -76,13 +76,18 @@ function install_composer_deps() {
                 sodium_version=$sodium_php72_php73_version
               fi
               fetch_package "$PHP_BASE_URL" "libsodium-${sodium_version}" "/app/vendor/libsodium" | indent
+            else
+              local ext_version=$(jq --raw-output ".require | .[\"ext-${ext}\"]" < "${BUILD_DIR}/composer.json")
+              install_pecl_extension $ext "$ext_version" "$CACHE_DIR" | indent
             fi
+
+            # TODO I don't think this instruction should be executed for all extensions
```

Comment:
I still have a question about this instruction. But the question also stands for the script version on `master`.. Do you know its purpose?

Reply:
I think this is more a question for @Soulou right ?